client/signup/steps/pressable-store/index.jsx

```diff
@@ -0,0 +1,115 @@
+import Gridicon from 'components/gridicon';
+
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+import HeroImage from './hero-image';
+
+export default React.createClass( {
+	displayName: 'PressableStoreStep',
+
+	propTypes: {
+		stepName: PropTypes.string.isRequired,
+		goToNextStep: PropTypes.func.isRequired,
+		signupDependencies: PropTypes.object.isRequired,
+	},
+
+	getInitialState() {
+		return {
+			email: '',
+			isValid: false,
+			error: null,
+		};
+	},
+
+	componentDidMount() {
+		if ( this._input ) {
+			this._input.focus();
+		}
+	},
+
+	onEmailChange( event ) {
+		const email = event.target.value;
+		const isValid = EmailValidator.validate( email );
+		const error = this.state.error && isValid
+			? null
+			: this.state.error;
+		this.setState( {
+			email,
+			isValid,
+			error,
+		} );
+	},
+
+	onSubmit( event ) {
+		event.preventDefault();
+
+		if ( ! this.state.isValid ) {
+			this.setState( {
+				error: this.translate( 'Please provide a valid email address.' ),
+			} );
+			return;
+		}
+
+		window.location.href = `https://my.pressable.com/signup/five-sites?email=${ encodeURIComponent( this.state.email ) }&wp-ecommerce=true`;
+	},
+
+	onEmailInputRef( input ) {
+		this._input = input;
+	},
+
+	renderStoreForm() {
+		return (
+			<div>
+				<LoggedOutForm className="pressable-store__form">
+					<HeroImage />
+
+					<FormSectionHeading className="pressable-store__heading">{ this.translate( 'Get your store for as low as $25 / month' ) }</FormSectionHeading>
+					<p className="pressable-store__copy">{ this.translate( 'We\'ve partnered with Pressable, a top-notch WordPress hosting provider, and WooCommerce, the go-to eCommerce solution for WordPress, to make setting up your store a snap.' ) }</p>
+
+					<LoggedOutFormFooter>
+						<FormLabel className="pressable-store__form-label" for="email">{ this.translate( 'Start by entering your email address:' ) }</FormLabel>
+						<div className="pressable-store__form-fields">
+							<FormTextInput ref={ this.onEmailInputRef } isError={ this.state.error } isValid={ this.state.isValid } onChange={ this.onEmailChange } className="pressable-store__form-email is-spaced" type="email" placeholder="Email Address" name="email" />
+							<FormButton onClick={ this.onSubmit } className="pressable-store__form-submit">{ this.translate( 'Get started on Pressable' ) } <Gridicon icon="external" size={ 12 } /></FormButton>
```

Comment:
Should we move `onClick={ this.onSubmit }` to L81 as an `onSubmit` event to account for non-clicking methods of submitting a form (like pressing enter)?